app/api_clients/jira_api_client.py

```diff
@@ -40,7 +40,7 @@ def _return_client(self, server, username, key):
 
     def initialize(self):
         server = environ.get('JIRA_SERVER_ADDRESS')
-        username = environ.get('JIRA_USERNAME')
+        username = environ.get('ADMIN_EMAIL')
```

Comment:
I don't think it will break, but since the dev/prod environments are different, I hope not :/